### PR TITLE
Add/Remove/Update metric names for PureFB v2.0.0

### LINKED ISSUE #2413: PureFB does not scrape 'clients' instruments
You can assign this to me, as I'll be updating the PureFB integration.

**Additional environment details (Operating System, Cloud provider, etc):**

Pure FlashBlade with any Datadog agent

**Steps to reproduce the issue:**
Add the /clients endpoint to the purefa datadog configuration 

**Describe the results you received:**

no metrics in Datadog

**Describe the results you expected:**

client metrics into datadog

````
purefb.clients.performance_average_bytes
purefb.clients.performance_bandwidth_bytes
purefb.clients.performance_latency_usec
purefb.clients.performance_throughput_iops
````

**Additional information you deem important (e.g. issue happens only occasionally):**

The metrics above are not listed correctly in https://github.com/DataDog/integrations-extras/blob/master/purefb/metadata.csv , and https://github.com/DataDog/integrations-extras/blob/master/purefb/datadog_checks/purefb/metrics.py - as they're under the array instrument, and not the clients instrument.

### PR DESCRIPTION
**Pull request by chrroberts-pure**
### What does this PR do?

The Pure Storage FlashBlade OpenMetrics Exporter has gone through some metrics changes, this PR is to update those metrics so Datadog ingests them correctly.

### Motivation

This PR is to ensure maximum compatibility and interoperability between the OpenMetrics Exporter and the Datadog agent.  

### Review checklist

- [ ] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [ ] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)
- [ ] If this PR includes a log pipeline, please add a description describing the remappers and processors. 

### Additional Notes

Anything else we should know when reviewing?

closes #2413

### CODE REVIEW

File: purefb/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,29 @@
 # CHANGELOG - PureFB
 
+## 1.0.5 / 2024-06-26
```
Comment (iliakur) [inline purefb/CHANGELOG.md:3]: If we're sticking to SemVer and we have an **added** section, this should be `1.1.0`

File: purefb/metadata.csv [hunk 2]
```diff
@@ -1,45 +1,45 @@
 metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric
-purefb.info,gauge,,,,FlashBlade system information,0,purefb,info,
```
Comment (iliakur) [inline purefb/metadata.csv:2]: Just make sure you're not removing any metrics but only moving them around. If you remove a metric, that's a breaking change and should be documented as such in the changelog/major version bump.

Comment (chrroberts-pure) [reply purefb/metadata.csv:2]: thanks for the feedback, i've updated to a major version 2.0.0 and added the requirements for the latest version of our OpenMetrics Exporter v1.0.11. 

Most of the metrics were just moved around to ensure that they were in alphabetical order, some were added/removed and noted in the changelog.

File: purefb/CHANGELOG.md (hunk 1 above)
Comment (chrroberts-pure) [reply purefb/CHANGELOG.md]: updated to 2.0.0

**Review by iliakur — APPROVED** [review]

File: purefb/metadata.csv (hunk 2 above)
Comment (iliakur) [reply purefb/metadata.csv:2]: sg thanks!

**Review by emarsha94 — APPROVED** [review]

**Review by bgoldberg122 — APPROVED** [review]

**Pull request merged**